.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.4
```

Comment:
@jasonoverby In general I'd want this in its own commit, since it's unrelated to the rest of the changes.